docs/en/query_language/functions/conditional_functions.md

```diff
@@ -1,11 +1,58 @@
 # Conditional functions
 
-## if(cond, then, else), cond ? operator then : else
+## if {#if}
 
-Returns `then` if `cond != 0`, or `else` if `cond = 0`.
-`cond` must be of type `UInt8`, and `then` and `else` must have the lowest common type.
+Evaluates condition `cond` and executes `then` expression if condition is met. Executes `else` otherwise.
 
-`then` and `else` can be `NULL`
+`if` - is a [ternary operation](https://en.wikipedia.org/wiki/Ternary_operation).
+
+**Syntax** 
+
+```sql
+SELECT if(cond, then, else)
+```
+
+**Parameters**
+
+- `cond` – The condition for evaluation. [UInt8](https://clickhouse.yandex/docs/ru/data_types/int_uint/).
+- `then` - The expression executed if condition is met. Any [data type](https://clickhouse.yandex/docs/en/data_types/).
+- `else` - The expression executed if condition is not met. Any [data type](https://clickhouse.yandex/docs/en/data_types/).
```

Comment:
All internal links should point to markdown files instead of pages where they are published